### DISCUSSION TITLE
Ask questions and get answers here - March 2021

### DISCUSSION THREAD

**Discussion by ghobona** [opengeospatial/developer-track#4] [Q&A]
To make it easier to post and respond to questions, we have set up a dedicated channel here.

**Comment by tomkralidis** [comment]
@ghobona is ogcmeet.org (https://files.gitter.im/5a9c2ff9d73408ce4f8fd299/eIj5/image.png) with https://github.com/opengeospatial/developer-track#agenda ?  For example, the agenda on GitHub shows Practical Work - Block 3 at 15:15-17:15, but on ogcmeet it is shown as 11:15 - 12:45 ?

Are the agenda and times (in EDT) on https://github.com/opengeospatial/developer-track#agenda authoritative?

cc @pvgenuchten @francbartoli

**Comment by ghobona** [comment]
The agenda on the GitHub repo is authoritative.
